### PR TITLE
Fix the labeled stats shown for labeled examples

### PR DESCRIPTION
**Pull request by rajasbansal**

### CODE REVIEW

File: src/autolabel/labeler.py [hunk 1]
```diff
@@ -127,7 +127,7 @@ def run(
         for current_index in track_with_stats(
             indices,
             postfix_dict,
-            total=len(inputs),
+            total=len(inputs) - current_index,
```
Comment (Abhinav-Naikawadi) [inline src/autolabel/labeler.py:130]: should this be "total=len(inputs) - current_index - 1"?

Comment (Abhinav-Naikawadi) [reply src/autolabel/labeler.py:130]: if current_index is the number of elements that have been completed so far?

Comment (rajasbansal) [reply src/autolabel/labeler.py:130]: Current index is the latest example which has *not* been labeled. So, if the first five examples have been labeled, the index would be 5 (0 indexing)

Comment (Abhinav-Naikawadi) [reply src/autolabel/labeler.py:130]: ah okay never mind then!

**Review by Abhinav-Naikawadi — APPROVED** [review]

**Pull request merged**